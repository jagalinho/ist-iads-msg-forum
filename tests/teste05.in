A 0 F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
L
A 0 KJ0WTBUA50K0FB08GKDSE6ULLV485U448PNFMRAT6GUH4O5YDRR5CS8IICIIKWUY3F4DGCY6KED7Z1GP0YVCSNJ64PZ1X120SFX5HPHVNGBADD8CQ6MZW08KZXAWECTEY0BRN2JF8URL
L
A 999 S
L
A 999 S
L
A 0 F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
L
A 999 KJ0WTBUA50K0FB08GKDSE6ULLV485U448PNFMRAT6GUH4O5YDRR5CS8IICIIKWUY3F4DGCY6KED7Z1GP0YVCSNJ64PZ1X120SFX5HPHVNGBADD8CQ6MZW08KZXAWECTEY0BRN2JF8URL
L
A 1 F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
L
A 0 S
L
A 999 S
L
A 0 KJ0WTBUA50K0FB08GKDSE6ULLV485U448PNFMRAT6GUH4O5YDRR5CS8IICIIKWUY3F4DGCY6KED7Z1GP0YVCSNJ64PZ1X120SFX5HPHVNGBADD8CQ6MZW08KZXAWECTEY0BRN2JF8URL
L
X
