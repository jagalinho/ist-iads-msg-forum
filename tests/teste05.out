*TOTAL MESSAGES:1
0:F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
*TOTAL MESSAGES:2
0:F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
0:KJ0WTBUA50K0FB08GKDSE6ULLV485U448PNFMRAT6GUH4O5YDRR5CS8IICIIKWUY3F4DGCY6KED7Z1GP0YVCSNJ64PZ1X120SFX5HPHVNGBADD8CQ6MZW08KZXAWECTEY0BRN2JF8URL
*TOTAL MESSAGES:3
0:F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
0:KJ0WTBUA50K0FB08GKDSE6ULLV485U448PNFMRAT6GUH4O5YDRR5CS8IICIIKWUY3F4DGCY6KED7Z1GP0YVCSNJ64PZ1X120SFX5HPHVNGBADD8CQ6MZW08KZXAWECTEY0BRN2JF8URL
999:S
*TOTAL MESSAGES:4
0:F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
0:KJ0WTBUA50K0FB08GKDSE6ULLV485U448PNFMRAT6GUH4O5YDRR5CS8IICIIKWUY3F4DGCY6KED7Z1GP0YVCSNJ64PZ1X120SFX5HPHVNGBADD8CQ6MZW08KZXAWECTEY0BRN2JF8URL
999:S
999:S
*TOTAL MESSAGES:5
0:F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
0:KJ0WTBUA50K0FB08GKDSE6ULLV485U448PNFMRAT6GUH4O5YDRR5CS8IICIIKWUY3F4DGCY6KED7Z1GP0YVCSNJ64PZ1X120SFX5HPHVNGBADD8CQ6MZW08KZXAWECTEY0BRN2JF8URL
999:S
999:S
0:F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
*TOTAL MESSAGES:6
0:F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
0:KJ0WTBUA50K0FB08GKDSE6ULLV485U448PNFMRAT6GUH4O5YDRR5CS8IICIIKWUY3F4DGCY6KED7Z1GP0YVCSNJ64PZ1X120SFX5HPHVNGBADD8CQ6MZW08KZXAWECTEY0BRN2JF8URL
999:S
999:S
0:F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
999:KJ0WTBUA50K0FB08GKDSE6ULLV485U448PNFMRAT6GUH4O5YDRR5CS8IICIIKWUY3F4DGCY6KED7Z1GP0YVCSNJ64PZ1X120SFX5HPHVNGBADD8CQ6MZW08KZXAWECTEY0BRN2JF8URL
*TOTAL MESSAGES:7
0:F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
0:KJ0WTBUA50K0FB08GKDSE6ULLV485U448PNFMRAT6GUH4O5YDRR5CS8IICIIKWUY3F4DGCY6KED7Z1GP0YVCSNJ64PZ1X120SFX5HPHVNGBADD8CQ6MZW08KZXAWECTEY0BRN2JF8URL
999:S
999:S
0:F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
999:KJ0WTBUA50K0FB08GKDSE6ULLV485U448PNFMRAT6GUH4O5YDRR5CS8IICIIKWUY3F4DGCY6KED7Z1GP0YVCSNJ64PZ1X120SFX5HPHVNGBADD8CQ6MZW08KZXAWECTEY0BRN2JF8URL
1:F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
*TOTAL MESSAGES:8
0:F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
0:KJ0WTBUA50K0FB08GKDSE6ULLV485U448PNFMRAT6GUH4O5YDRR5CS8IICIIKWUY3F4DGCY6KED7Z1GP0YVCSNJ64PZ1X120SFX5HPHVNGBADD8CQ6MZW08KZXAWECTEY0BRN2JF8URL
999:S
999:S
0:F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
999:KJ0WTBUA50K0FB08GKDSE6ULLV485U448PNFMRAT6GUH4O5YDRR5CS8IICIIKWUY3F4DGCY6KED7Z1GP0YVCSNJ64PZ1X120SFX5HPHVNGBADD8CQ6MZW08KZXAWECTEY0BRN2JF8URL
1:F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
0:S
*TOTAL MESSAGES:9
0:F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
0:KJ0WTBUA50K0FB08GKDSE6ULLV485U448PNFMRAT6GUH4O5YDRR5CS8IICIIKWUY3F4DGCY6KED7Z1GP0YVCSNJ64PZ1X120SFX5HPHVNGBADD8CQ6MZW08KZXAWECTEY0BRN2JF8URL
999:S
999:S
0:F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
999:KJ0WTBUA50K0FB08GKDSE6ULLV485U448PNFMRAT6GUH4O5YDRR5CS8IICIIKWUY3F4DGCY6KED7Z1GP0YVCSNJ64PZ1X120SFX5HPHVNGBADD8CQ6MZW08KZXAWECTEY0BRN2JF8URL
1:F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
0:S
999:S
*TOTAL MESSAGES:10
0:F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
0:KJ0WTBUA50K0FB08GKDSE6ULLV485U448PNFMRAT6GUH4O5YDRR5CS8IICIIKWUY3F4DGCY6KED7Z1GP0YVCSNJ64PZ1X120SFX5HPHVNGBADD8CQ6MZW08KZXAWECTEY0BRN2JF8URL
999:S
999:S
0:F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
999:KJ0WTBUA50K0FB08GKDSE6ULLV485U448PNFMRAT6GUH4O5YDRR5CS8IICIIKWUY3F4DGCY6KED7Z1GP0YVCSNJ64PZ1X120SFX5HPHVNGBADD8CQ6MZW08KZXAWECTEY0BRN2JF8URL
1:F0GIZ1YTINSLGH7DYXOMI8NCI7WMIK70CII1JZO37S7NO3CPKWZOOVUZM5B5G9G5XEIPQ07X6AVI4STV64K0Z6U24UK7FGL24NEIINNQBIOOQFTOTA0CR3JJ9P00NM4AW3DHP7Z5NHI0
0:S
999:S
0:KJ0WTBUA50K0FB08GKDSE6ULLV485U448PNFMRAT6GUH4O5YDRR5CS8IICIIKWUY3F4DGCY6KED7Z1GP0YVCSNJ64PZ1X120SFX5HPHVNGBADD8CQ6MZW08KZXAWECTEY0BRN2JF8URL
10
